OrbitLinuxTracing/include/OrbitLinuxTracing/TracingOptions.h

```diff
@@ -0,0 +1,21 @@
+#ifndef ORBITLINUXTRACING_INCLUDE_ORBITLINUXTRACING_TRACINGOPTIONS_H_
+#define ORBITLINUXTRACING_INCLUDE_ORBITLINUXTRACING_TRACINGOPTIONS_H_
+
+namespace LinuxTracing {
+enum UnwindingMethod {
+  kUndefined = 0,
+  kFramePointers,
+  kDwarf
+};
+
+struct TracingOptions {
+  bool trace_context_switches_ = true;
```

Comment:
Remove the trailing underscores as there are public.

Reply:
Will do so.